styling fixes & countdown voting clock

lazarovicedo opened the pull request:
keep footer on the bottom, added loading bar (vertical centered) on token list, truncate urls on representatives list, display wallet version on the footer (from data stored on package.json), truncate token name on send page

lazarovicedo committed:
no message

lazarovicedo committed:
no message

lazarovicedo committed:
Merge branch 'master' of https://github.com/lazarovicedo/wallet-web

* 'master' of https://github.com/lazarovicedo/wallet-web:
  hide my transaction link

lazarovicedo committed:
keep footer on the bottom, added loading bar (vertical centered) on token list, truncate urls on representatives list, display wallet version on the footer (from data stored on package.json), truncate token name on send page

lazarovicedo committed:
Merge branch 'master' into master

lazarovicedo committed:
no login captcha file

DevObs1 reviewed (commented):
Hi @lazarovicedo,
The sitekey corresponds to the domain tronscan.org?
src/components/account/LoginWithCaptcha.js, line 108

lazarovicedo commented:
DevObs1, no. that sitekey is for localhost. Anyway, that captcha feature will be disabled by now as suggested by @Rovak

DevObs1 commented:
Why should it be disabled?
Because I wanted to work on this fix too :) but I was thinking of contacting @Rovak Rovak before to generate a sitekey for PR

lazarovicedo committed:
On account view, removed view password button for trustness.
Hide token table header if the are tokens to list. Removed repeated alert message regarding testnet

lazarovicedo committed:
formatted number of votes for easy reading. Load bar for list of representatives.

lazarovicedo commented:
@DevObs1 , Do you know any popular web wallet that implements a captcha?

DevObs1 commented:
@lazarovicedo : I looked in a dozen sites but none offers reCaptcha (either for login or registration)

lazarovicedo committed:
help info updated on voting page and count clock added (every 6h UTC)

lazarovicedo committed:
fixes and cleanup on votes countdown clock

lazarovicedo committed:
Merge branch 'master' into master

lazarovicedo committed:
Merge branch 'master' into master

lazarovicedo committed:
function not needed anymore

lazarovicedo committed:
Merge branch 'master' of https://github.com/lazarovicedo/wallet-web

* 'master' of https://github.com/lazarovicedo/wallet-web: (26 commits)
  fix issue #178
  Change password to private key
  change rendering condition
  apply pagination for accounts
  add PaginationBar component
  remove unused import
  change comparison operator
  add translate
  add translate
  remove duplicate key
  bump wallet-api
  bump wallet-api
  bump wallet-api
  bump wallet-api
  fix addresses
  resolve conflict
  Resolve conflict
  del lock
  Update ko.js
  Update ko.js
  ...

# Conflicts:
#	src/components/account/Account.js

lazarovicedo committed:
no message

lazarovicedo committed:
no message

lazarovicedo committed:
removes grey bar on the bottom of the login page.

lazarovicedo committed:
Merge branch 'master' into master

lazarovicedo committed:
fixes lost logged-in session when reloading the page

lazarovicedo committed:
no tokens found message instead of an empty table with headers

Rovak reviewed (commented):
I get this error when registering a new account

![screenshot from 2018-05-11 19-45-26](https://user-images.githubusercontent.com/376155/39938637-e3d46540-5553-11e8-91a5-3e9356608bbc.png)

Rovak commented:
After registering / logging in and refreshing i get this error 

![screenshot from 2018-05-11 19-46-17](https://user-images.githubusercontent.com/376155/39938703-17dec736-5554-11e8-9c1a-6763d0d663fb.png)

Rovak commented on src/reducers/app.js:
```diff
@@ -20,11 +20,11 @@ const initialState = {
-    key: undefined,
-    address: undefined,
-    isLoggedIn: false,
+  activeLanguage: (Lockr.get("language") !== undefined ? Lockr.get("language") : 'en'),
```
`Locker.get` has a second parameter for a default value, this can be simplified  to `Lockr.get("language", "en")`

Rovak commented on src/reducers/app.js:
```diff
@@ -20,11 +20,11 @@ const initialState = {
-    isLoggedIn: false,
+  activeLanguage: (Lockr.get("language") !== undefined ? Lockr.get("language") : 'en'),
+  account: {   
+    key: (Lockr.get("account_key") !== undefined ? Lockr.get("account_key") : undefined),
```
can be simplified to `Lockr.get("account_key")`

Rovak commented on src/reducers/app.js:
```diff
@@ -20,11 +20,11 @@ const initialState = {
+  account: {   
+    key: (Lockr.get("account_key") !== undefined ? Lockr.get("account_key") : undefined),
+    address: (Lockr.get("account_key") !== undefined ? privateKeyToAddress(Lockr.get("account_key")) : undefined),
+    isLoggedIn: (Lockr.get("account_key") !== undefined ? true : false)
```
can be simplified to `Lockr.get("account_key") !== undefined`

lazarovicedo committed:
Update app.js

lazarovicedo committed:
Update app.js

lazarovicedo reviewed (commented):
simplified to Lockr.get("account_key")

lazarovicedo committed:
Merge branch 'master' into master

lazarovicedo committed:
Update app.js

lazarovicedo committed:
removed componentDidMount duplicated  in login,
fix var reasignment
mediaquery not used on representatives.js cause now using responsive table
removed duplicated send_TRX word

lazarovicedo committed:
no message

lazarovicedo committed:
added testnet warning on receive , added testnet text n footer, added IS_TESTNET const for global testnet alert management

lazarovicedo committed:
added FormattedMessage function for better paragraph formatting.
Updated info on voting intervals (6h) when applying for delegate

Rovak commented:
very nice improvements, thanks!
